pageserver/src/tenant/timeline.rs

```diff
@@ -4155,6 +4161,169 @@ impl Timeline {
         false
     }
 
+    async fn create_image_layers_for_rel_blocks(
```

Comment:
it's only creating up to one image layer here, right? it is called `create_image_layers_for_rel_blocks` however.

The returned Vec has up to one item, so maybe return an option instead? the `extend` function should support this as well.